### PR TITLE
Added "Fake Fake Teleport" effect

### PR DESCRIPTION
**Pull request by OnlyRealNubs**
Implements #1218

### CODE REVIEW

File: ChaosMod/Effects/db/Player/PlayerTpController.cpp [hunk 1]
```diff
@@ -386,4 +386,75 @@ REGISTER_EFFECT(OnStartFakeTp, nullptr, nullptr, EffectInfo
...
+	auto overrideId                       = selectedLocationInfo.type;
+	GetComponent<EffectDispatcher>()->OverrideEffectNameId("tp_fakex2", overrideId);
+
+	Player player     = PLAYER_ID();
+	Ped playerPed     = PLAYER_PED_ID();
+	Vehicle playerVeh = IS_PED_IN_ANY_VEHICLE(playerPed, false) ? GET_VEHICLE_PED_IS_IN(playerPed, false) : 0;
+
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	Hooks::EnableScriptThreadBlock();
```
Comment (Reguas) [inline ChaosMod/Effects/db/Player/PlayerTpController.cpp:404]: Script thread block enables, but never disables

**Review by Reguas — CHANGES_REQUESTED** [review]

File: ChaosMod/Effects/db/Player/PlayerTpController.cpp (hunk 1 above)
Comment (OnlyRealNubs) [reply ChaosMod/Effects/db/Player/PlayerTpController.cpp:404]: Whoops! Sometimes I forget things, that should be fixed in the latest commit.

File: ConfigApp/Effects.cs [hunk 2]
```diff
@@ -376,6 +376,7 @@ public enum EffectTimedType
             { "peds_hotcougars", new EffectInfo("Hot Cougars In Your Area", EffectCategory.Peds, true, true) },          
             { "peds_grapple_guns", new EffectInfo("Gravity Guns", EffectCategory.Peds, true) },
             { "timecycle_darkworld", new EffectInfo("A Dark World", EffectCategory.Screen, true, true) },
+            { "tp_fakex2", new EffectInfo("Fake Fake Teleport", EffectCategory.Player, false) },
```
Comment (pongo1231) [inline ConfigApp/Effects.cs:379]:
```suggestion
            { "tp_fakex2", new EffectInfo("Fake Fake Teleport", EffectCategory.Player) },
```

Pedantic but for consistency purposes

**Comment by OnlyRealNubs** [comment]
Resolved.

**Pull request merged**